UML/TapiPhotonicMedia.uml

```diff
@@ -889,7 +854,7 @@ Any combination of frequency slots is allowed as long as no two frequency slots
   <OpenModel_Profile:Experimental xmi:id="_NG-4EHNoEeiPPoPDo3q5jA" base_Element="_YS8DkIfeEeet-8tHdGGp_w"/>
   <OpenModel_Profile:Experimental xmi:id="_OQvVcHNoEeiPPoPDo3q5jA" base_Element="_VvSIYEUIEead1bezhJG4aw"/>
   <OpenInterfaceModel_Profile:OpenInterfaceModelAttribute xmi:id="_TnDTYXPXEeiPPoPDo3q5jA" base_Property="_TnDTYHPXEeiPPoPDo3q5jA" attributeValueChangeNotification="YES" bitLength="LENGTH_64_BIT"/>
-  <OpenModel_Profile:OpenModelAttribute xmi:id="_TnDTYnPXEeiPPoPDo3q5jA" base_StructuralFeature="_TnDTYHPXEeiPPoPDo3q5jA" valueRange="193.00013..195.99988" unit="THz"/>
+  <OpenModel_Profile:OpenModelAttribute xmi:id="_TnDTYnPXEeiPPoPDo3q5jA" base_StructuralFeature="_TnDTYHPXEeiPPoPDo3q5jA" valueRange="193000..195999" unit="THz"/>

```

Comment:
To my knowledge the range for frequencies is: 191.7 THz to 196.1 -> 191700000 ... 196100000 MHz.

Reply:
will remove the value ranges as these need discussion/consensus